doc/releases/changelog-dev.md

```diff
@@ -189,6 +189,9 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* Fix a bug where trainable parameters in the post-processing of finite diff were incorrect for Jax.
```

Comment:
Should we mention that this is only when using the transform on QNodes manually?